packages/ui/src/components/pages/Accounts/AccountControls/index.tsx

```diff
@@ -43,8 +49,8 @@ const AccountControls: FC<AccountControlsProps> = ({ className = '', account })
         <MoreVertIcon />
       </IconButton>
       <Menu open={open} anchorEl={anchorEl} onClose={doClose}>
-        {AccountControlsOptions.map(({ action, event }) => (
-          <MenuItem key={action} onClick={() => openDialogOnAction(event)}>
+        {AccountControlsOptions.map(({ action, event, className = '' }) => (
+          <MenuItem key={action} onClick={() => openDialogOnAction(event)} className={className}>
```

Comment:
I'm thinking of adding icons to each menu item here to make it less boring, what do you think? MUI does support icons for menu items, so this would make it an easy tweak.

We could do:
1. Rename: https://mui.com/material-ui/material-icons/?query=rename&selected=DriveFileRenameOutline
2. Delete: https://mui.com/material-ui/material-icons/?query=Remove&selected=Delete
3. Show Address QR Code: https://mui.com/material-ui/material-icons/?query=Qr+code&selected=QrCode